week10/diaryapp/src/App.jsx

```diff
@@ -0,0 +1,18 @@
+import NoteListPage from "./pages/NoteListPage";
+import WriteNotePage from "./pages/WriteNotePage";
+import NoteDetailPage from "./pages/NoteDetailPage";
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+
+function App() {
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route path="/detail/:noteDate" element={<NoteDetailPage/>} />
```

Comment:
<Route path="/detail/:date" element={<NoteDetailPage/>} />
```suggestion
        <Route path="/detail/:date" element={<NoteDetailPage/>} />
```